### ISSUE TITLE
No guide on how to change new projects from V2 to V3

### ISSUE THREAD

**Issue by tdmac-consulting** [statamic/docs#756]
After working on a legacy project using v2 now all new projects are v2 not v3, there is no guide on how to resolve this issue, or how to choose/force what version of statamic to be used when initiating a new project like `statamic new my_site -v3` for example.

**Comment by duncanmcclean** [comment]
You'll need to update the Statamic CLI to support v3. This command should do the job:

```
composer global update statamic/cli^2.4.0
```

**Comment by tdmac-consulting** [comment]
So that didn't work. in the end I had to edit the global composer files.
`~/ composer`
`composer ls la`
`rm composer.lock`
`nano composer.json`
Once the editor was open i changed the version of the statamic manually from **1.0.0** to **2.4.0**. I then ran:
`composer install`

Then I could use `composer global update statamic/cli:^2.4.0`

**Comment by Konafets** [comment]
What do you mean with "it didn't" work? Did you recognise the _global_ flag in @duncanmcclean answer?

**Comment by tdmac-consulting** [comment]
Yes, I copied and pasted his answer directly into the command line, but was met with an error. Hence, I had to complete the process in the manner described.

Unfortunately, I did not take note of the error message at the time, as I was being guided on a zoom call by a senior developer on how to resolve the issue. I just thought I should retrospectively document my process for anyone else, should they experience the same issue as I did when trying to use the solution provided by @duncanmcclean

**Comment by Konafets** [comment]
I think it worked updating the _global_ statamic install but I think you have it also locally (in your project folder) installed. Its my assumption as you edit the local `composer.lock` file and calling `composer install` in your folder.
